manim/config.py

```diff
@@ -23,67 +23,72 @@ def parse_cli():
             "-p", "--preview",
             action="store_true",
             help="Automatically open the saved file once its done",
-        ),
+        )
         parser.add_argument(
             "-w", "--write_to_movie",
             action="store_true",
             help="Render the scene as a movie file",
-        ),
+        )
         parser.add_argument(
             "-s", "--save_last_frame",
             action="store_true",
             help="Save the last frame",
-        ),
+        )
+        parser.add_argument(
+            "--dry_run", 
+            action="store_true",
+            help= "run a dry test",
```

Comment:
Could yo please change "run a dry test" to "run a dry test (render scenes but generate no output files)" or some such?

Reply:
Done !